standards/bulk.md

```diff
@@ -0,0 +1,134 @@
+# Bulk Operations
+
+## Overview
```

Comment:
Should the standards have an opinion on transnationality of the bulk operation?  For instance if we wanted to create all or none of the resources in the bulk request.

Reply:
That is a very good question! I think there should be an option for that. But some of that is dependent on the answer to how we treat the atomicity of the PATCH request in general. If we move ahead with that direct interpretation than its implied here, no such thing as partial success. All or nothing from a transaction point of view.

Reply:
Another consideration might be how we deal with that transaction across multiple bulk requests.

Reply:
Yep to clarify we need to answer too questions here:
1. Is a bulk PATCH request atomic, and all operations act as a single transaction to succeed or fail together? (inline with expected PATCH RFC). 
2. Is bulk  PATCH request atomicity a setting that can be passed as a parameter if needed?
3. Do we need a method to handle "paging" bulk requests that require atomicity across multiple requests (multiple requests as a single transaction)?

Reply:
1. Added parameter and default to control atomicity vs isolated transactions.
2. Added.
3. No paging between requests and no atomicity between requests... its out of scope.